packages/terra-dialog-modal/CHANGELOG.md

```diff
@@ -91,8 +94,8 @@
 * Added
   * Added clinical-lowlight-theme theming files
 
-* Changed
-  * Update docs to use mdx syntax.
+  * Changed
+    * Update docs to use mdx syntax.
```

Comment:
I don't think this should have the indention.

Reply:
fixed: https://github.com/cerner/terra-framework/pull/1105/commits/abb5364c1fa144602adab79f0c3c397faffab3f6